addons/point_of_sale/static/src/js/models.js

```diff
@@ -2669,6 +2673,16 @@ export class Order extends PosModel {
         }
         return json;
     }
+    export_shipping_date_for_printing() {
```

Comment:
Prefer camelCase again and add `_` in front of the method since you're not using it outside (private)